src/queries/newsItems.js

```diff
@@ -8,6 +8,7 @@ export const GET_NEWS_ITEMS = gql`
     $dataProvider: String
     $dataProviderId: ID
     $excludeDataProviderIds: [ID]
+    $excludeMowasRegionalKeys: [ID]
```

Comment:
please change this to `[String]`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1063/commits/f0667f51f883a7478cddd9ef80edd61b736d3f45